resources/lang/vi/passwords.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Password Reset Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are the default lines which match reasons
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset'     => 'Mật khẩu mới đã được cập nhật!',
+    'sent'      => 'Hướng dẫn cấp lại mật khẩu đã được gửi!',
+    'throttled' => 'Vui lòng đợi trước khi thử lại.',
+    'token'     => 'Mã khôi phục mật khẩu không hợp lệ.',
+    'user'      => 'Không tìm thấy người dùng với địa chỉ email này.',
```

Comment:
```suggestion
    'reset' => 'Mật khẩu mới đã được cập nhật!',
    'sent' => 'Hướng dẫn cấp lại mật khẩu đã được gửi!',
    'throttled' => 'Vui lòng đợi trước khi thử lại.',
    'token' => 'Mã khôi phục mật khẩu không hợp lệ.',
    'user' => 'Không tìm thấy người dùng với địa chỉ email này.',
```
Sửa lại cho m mấy chỗ này nhé cho đúng convention nhé